src/portable/portable.c

```diff
@@ -565,16 +569,24 @@ static int extract_image_and_extensions(
                 r = parse_env_file(f, os_release->name,
                                    "ID", &id,
                                    "VERSION_ID", &version_id,
-                                   "SYSEXT_LEVEL", &sysext_level);
+                                   "SYSEXT_LEVEL", &sysext_level,
+                                   "PORTABLE_PREFIXES", &prefixes);
                 if (r < 0)
                         return r;
+
+                if (prefixes) {
```

Comment:
as the comment above the block says, right now this is used for validation purposes, but this adds to that, so the comment should be updated to reflect that